analytics_dashboard/core/middleware.py

```diff
@@ -8,10 +8,24 @@
 from django.template.response import TemplateResponse
 
 from core.exceptions import ServiceUnavailableError
+from waffle import flag_is_active
 
 logger = logging.getLogger(__name__)
 
 
+# For https://tasks.opencraft.com/browse/SE-3950 . If an upstream solution is invented by the time you're rebasing this,
+# remove it and the line that includes it from the middleware settings.
+class LanguageHeaderPreemptMiddleware(object):
+    """
+    If the appropriate waffle flag is present, ignore the 'Accept-Language' header.
+    """
+    def process_request(self, request):
+        if not flag_is_active(request, 'ignore_accept_language'):
+            return
+        if request.META.has_key('HTTP_ACCEPT_LANGUAGE'):
```

Comment:
Small nit pick: new line before this line please

```suggestion

        if request.META.has_key('HTTP_ACCEPT_LANGUAGE'):
```